models/zeroex/optimism/zeroex_api_optimism_fills.sql

```diff
@@ -363,4 +371,7 @@ AND mp.minute >= date_trunc('day', now() - interval '1 week')
 {% endif %}
 {% if not is_incremental() %}
 AND mp.minute >= '{{zeroex_v3_start_date}}'
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+LEFT OUTER JOIN {{ ref('tokens_ethereum_erc20') }} ts ON ts.contract_address = taker_token
+LEFT OUTER JOIN {{ ref('tokens_ethereum_erc20') }} ms ON ms.contract_address = maker_token
```

Comment:
Shouldn't we be using 'tokens_optimism_erc20' instead of 'tokens_ethereum_erc20' here, @RantumBits ?